test/unit/core/kernel/persistence/OntologyRdfTest.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2018 (original work) Open Assessment Technologies SA;
+ *
+ */
+
+namespace oat\generis\test\unit\core\kernel\persistence;
+
+use core_kernel_classes_Triple;
+use oat\generis\model\data\event\ResourceCreated;
+use oat\generis\model\data\Ontology;
+use oat\generis\model\OntologyRdf;
+use oat\generis\model\OntologyRdfs;
+use oat\generis\test\GenerisTestCase;
+use oat\oatbox\action\Action;
+use oat\oatbox\event\EventManager;
+use oat\oatbox\event\GenericEvent;
+use Prophecy\Argument;
+use Prophecy\Prediction\CallTimesPrediction;
+
+/**
+ *
+ */
+class OntologyRdfTest extends GenerisTestCase
+{
+    /**
+     * @dataProvider getOntologies
+     */
+    public function testRdfInterface(Ontology $ontology)
+    {
+        $this->assertInstanceOf(Ontology::class, $ontology);
```

Comment:
You already ensure that this variable is instance of Ontology by using parameter type hint